packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -1371,20 +1390,12 @@ export function makeLiveSlots(
     vatParameters,
     gcTools,
     liveSlotsConsole,
+    buildVatNamespace,
   );
-  const {
-    vatGlobals,
-    inescapableGlobalProperties,
-    dispatch,
-    setBuildRootObject,
-    possiblyDeadSet,
-    testHooks,
-  } = r; // omit 'm'
+  const { dispatch, startVat, possiblyDeadSet, testHooks } = r; // omit 'm'
```

Comment:
oh, hey, we aren't returning `vatGlobals` or `inescapableGlobalProperties` anyways, so we should definitely make that change in `manager-local.js`, they'll always be empty